user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -166,6 +166,22 @@ const handler = getLambdaHandler(myContract)(async event => {
 });
 ```
 
+If you use your lambda as an ApiGateway integration, you will need to use middlewares to parse the string body, validate the input and output formats and serialize the output body. You may also need to handler error cases.
```

Comment:
We can add this in a dedicated section like

```ts
### Validate the input and output at runtime
```

Reply:
Documentation text updated, in a new section